sdk/core/azure-core/CHANGELOG.md

```diff
@@ -1,14 +1,10 @@
 # Release History
 
-## 1.29.2 (Unreleased)
-
-### Features Added
-
-### Breaking Changes
+## 1.29.2 (2023-08-14)
 
 ### Bugs Fixed
 
-### Other Changes
+- Added a default implementation for `AsyncTokenCredential.__aexit__()` #31573
```

Comment:
Do you think we should include the version bump?

Reply:
I want to use 1.29.2 to override 1.29.1. :)

Reply:
Sorry, I meant the typing-extensions version bump